### ISSUE TITLE
Links missing in venue  page

### ISSUE THREAD

**Issue by ChillarAnand** [pythonindia/inpycon2015#21]
![ven](https://cloud.githubusercontent.com/assets/4463796/9428263/671a6b00-49c4-11e5-8994-8085e5b701d0.png)

All three buttons doesn't have a link. 
Add relevant links or remove buttons?

**Comment by kracekumar** [comment]
Yes, these were all last year icons. We will add them as we go close to
event.

On Sun, Aug 23, 2015 at 6:27 PM, Anand Reddy Pandikunta <
notifications@github.com> wrote:

> [image: ven]
> https://cloud.githubusercontent.com/assets/4463796/9428263/671a6b00-49c4-11e5-8994-8085e5b701d0.png
> 
> All three buttons doesn't have a link.
> Add relevant links or remove buttons?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/pythonindia/inpycon2015/issues/21.

## 

Regards
Kracekumar Ramaraju
http://kracekumar.com
+91 85530 29521
